server.rb

```diff
@@ -11,10 +11,12 @@
 require 'webrick'
 require 'json'
 
-puts 'Server started: http://localhost:3000/'
+port = ENV['PORT'].nil? ? 3000 : ENV['PORT'].to_i
+
+puts "Server started: http://localhost:#{port}/"
```

Comment:
Can you do the same for the `go` server and make sure it outputs the right port?